modules/apps/commerce/commerce-order-content-web/src/main/resources/META-INF/resources/js/InfoBoxModalNotes.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import ClayForm from '@clayui/form';
+import ClayModal from '@clayui/modal';
+import React from 'react';
+
+import InfoBoxModalNotesInput from './info_box/modal/InfoBoxModalNotesInput';
+
+const InfoBoxModalNotes = ({
+	handleDelete,
+	handleSubmit,
+	handleToggle,
+	id,
+	isRestricted,
+	isValid,
+	label,
+	notes,
+	observer,
+	onOpenChange,
+	open,
+	setInputValue,
+	spritemap,
+}) => {
+	const InputRenderer = InfoBoxModalNotesInput;
```

Comment:
Non-blocking: this abstraction is no more needed, you could simply include the `InfoBoxModalNotesInput` within this Modal.